SS/js/ArgyleEngine.ts

```diff
@@ -167,7 +167,7 @@ function gridToScreen(xOrVect: any, y?: number): Vector2 {
     if (typeof xOrVect != "number")
         out = new Vector2((600 + 32 * xOrVect.x - 32 * xOrVect.y) - 87, (16 * xOrVect.x + 16 * xOrVect.y) + 44);
     else 
-        out = new Vector2((600 + 32 * xOrVect - 32 * y) - 87, (16 * xOrVect + 16 * y) + 44);
+        out = new Vector2((600 + 32 * xOrVect - 32 * (y || 0)) - 87, (16 * xOrVect + 16 * (y || 0)) + 44);
```

Comment:
I suppose we could simply enforce a vector and then we wouldn't need to worry about passing a y or not. Could also do something like `y: number = 0` so that intellisense can tell the user to set a y if they need it. If we did want to go back and make this more of a standalone engine, we might want to consider such things. As is, I mostly just wanted to get it set up for easy running.  Personally, I think it's fine for now but you're welcome to jump in and make changes if you want!